assets/css/style.css

```diff
@@ -74,63 +206,68 @@ main {
 }
 
 .search-bar {
-  position: relative;
+  display: flex;
```

Comment:
I like the way you change the position of the magnifying glass. Much simpler than the way I did!

Reply:
Ty 😀